components/infobox/extensions/commons/years_active_base.lua

```diff
@@ -115,29 +115,16 @@ function ActiveYears._calculate(conditions)
 	for index, year in ipairs(sortedYears) do
 		if index == 1 then
 			startYear = year
-			endYear = year
-		else
-			if year - endYear == 1 then
-				endYear = year
-			else
-				if startYear == endYear then
-					table.insert(yearRanges, tostring(startYear))
-				else
-					table.insert(yearRanges, tostring(startYear) .. ' - ' .. tostring(endYear))
-				end
-				startYear = year
-				endYear = year
-			end
+		elseif year - endYear ~= 1 then
```

Comment:
> 1, since the data must be increasing in order

Reply:
you mean `> 1`, right?
(it converted the `>` into a quote)